modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/internal/upgrade/v4_3_0/UpgradeKBArticle.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.knowledge.base.internal.upgrade.v4_3_0;
+
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+/**
+ * @author Alicia García
+ */
+public class UpgradeKBArticle extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		if (!hasColumn("KBArticle", "expirationDate")) {
+			alterTableAddColumn("KBArticle", "expirationDate", "DATE null");
+		}
+
+		if (!hasColumn("KBArticle", "reviewDate")) {
+			alterTableAddColumn("KBArticle", "reviewDate", "DATE null");
+		}
```

Comment:
Not necessary for this pr to be forwarded, but it is no longer necessary to check if the column exists. See https://liferay.atlassian.net/wiki/spaces/ENGCOREUPGRADE/blog/2022/08/25/2108196950/Simplify+DB+modification+framework+Part+2

Reply:
yeah, I realized after that and made an additional commit on [04c6c6a](https://github.com/liferay-lima/liferay-portal/pull/3550/commits/04c6c6a2c5c151e4df13ef8e1cdaa65bef365ead) deleting this class